crates/astria-composer/src/searcher/collector.rs

```diff
@@ -213,12 +213,11 @@ impl Collector {
 /// TODO(https://github.com/astriaorg/astria/issues/216): add timeouts for
 /// `subscribe_full_pendings_txs` (more complex because it's a stream).
 #[derive(Clone, Debug)]
-struct EthClient {
+struct GethClient {
```

Comment:
why is this geth now? what geth-specific functionality are we using?

Reply:
it depends on the websockets endpoint, which is geth specific. nothing changed, just renaming for clarity